x-pack/plugins/ml/public/application/explorer/explorer.js

```diff
@@ -302,6 +303,34 @@ export class Explorer extends React.Component {
               setSelectedCells={this.props.setSelectedCells}
             />
             <EuiSpacer size="m" />
+            {annotationsError !== null && (
+              <>
+                <EuiTitle className="panel-title">
+                  <h2>
+                    <FormattedMessage
+                      id="xpack.ml.explorer.annotationsErrorTitle"
+                      defaultMessage="Annotations"
+                    />
+                  </h2>
+                </EuiTitle>
+                <EuiPanel>
+                  <EuiCallOut
+                    title={i18n.translate('xpack.ml.explorer.annotationsErrorCallOutTitle', {
+                      defaultMessage: 'An error occurred loading annotations:',
+                    })}
+                    color="danger"
+                    iconType="alert"
+                  >
+                    <p>
+                      {typeof annotationsError === 'string'
+                        ? annotationsError
+                        : JSON.stringify(annotationsError)}
```

Comment:
I think it might be worth to use `extractErrorMessage` here to handle different error response shapes

Reply:
Good idea, fixed in 83ffd4b.